app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -85,6 +85,18 @@ def clean_json(self):
                     field="json",
                 )
 
+        for viewport in [v for v in value if "slice_plane_indicator" in v]:

```

Comment:
The list comprehension in a for loop is confusing, could you move the filter to the main loop?